CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+#ECCN:Open Source 5D002
+
+#GUSINFO:TUA-Admin-Intelligent App Dev Platform,Wave Developer Experience - Analytics
+* @salesforcecli/analytics-cli
+# Techical writers will be added as reviewers on markdown changes.
```

Comment:
typo ? Technical

Reply:
Ha, good catch. Someone else wrote that comment awhile ago. I'm going to leave it for posterity's sake 😄